deploy/1-eks-cluster-with-new-vpc/main.tf

```diff
@@ -142,4 +165,6 @@ module "kubernetes-addons" {
   enable_aws_load_balancer_controller = true
   enable_metrics_server               = true
   enable_cluster_autoscaler           = true
+
+  depends_on = [module.aws-eks-accelerator-for-terraform]
```

Comment:
@askulkarni2 We may need to change this to this line based in our recent timing issues. `depends_on = [module.aws-eks-accelerator-for-terraform.managed_node_groups]`